stix2/test/v20/test_observed_data.py

```diff
@@ -1140,15 +1140,6 @@ def test_process_example_windows_process_ext_empty():
     assert excinfo.value.properties == sorted(properties_of_extension)
 
 
-def test_process_example_extensions_empty():
```

Comment:
This test could be re-written to verify the serialization if an empty dictionary is set purposely.